src/main/java/org/dataloader/DataLoaderRegistry.java

```diff
@@ -144,6 +153,9 @@ private static DataLoaderOptions setInInstrumentation(DataLoaderOptions options,
      */
     public DataLoaderRegistry register(DataLoader<?, ?> dataLoader) {
         String name = Assertions.nonNull(dataLoader.getName(), () -> "The DataLoader must have a non null name");
+        if (strictMode) {
+            assertKeyStrictly(name);
+        }
         dataLoaders.put(name, nameAndInstrumentDL(name, instrumentation, dataLoader));
```

Comment:
You need to cover `org.dataloader.DataLoaderRegistry#computeIfAbsent` as well

I think this code would be better put in the `nameAndInstrumentDL` method as a common place

The following is all the places it is called



```
instrumentDLs(Map<String, DataLoader<?, ?>>, DataLoaderInstrumentation)
register(DataLoader<?, ?>)
register(String, DataLoader<?, ?>)
registerAndGet(String, DataLoader<?, ?>)
computeIfAbsent(String, Function<String, DataLoader<?, ?>>)
```